routers/router.php

```diff
@@ -166,6 +169,12 @@ private function build_commands($command, $parameter) {
         }
         return $this->build_aspath_regexp($parameter);
 
+      case 'as-path-regex6':
```

Comment:
Not required if the IPv6 specific AS path regexp function is removed and its logic included in the existing AS path function.